extensions/amp-story-auto-ads/0.1/test/test-story-ad-config.js

```diff
@@ -204,4 +204,72 @@ describes.realWin('amp-story-auto-ads:config', {amp: true}, (env) => {
       });
     });
   });
+
+  it('Does use remote config when src attribute is provided', async () => {
```

Comment:
nit: lets make everything lowercase always here and below to be consistent with the rest of the codebase